### PR TITLE
Solved Issue #2206

### LINKED ISSUE #2206: [Feature] primal.net on nostr section
### Feature description

Add https://primal.net/ on Nostr section

### Describe the solution

Add https://primal.net/ on Nostr section

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by Tejasp2003**
### Link this PR to an issue [optional]

Fixes #2206

### CODE REVIEW

File: src/app/screens/Discover/websites.json [hunk 1]
```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
+        "logo": "https://primal.net/assets/logo_fire-409917ad.svg",
```
Comment (escapedcat) [inline src/app/screens/Discover/websites.json]: You can use this one now:
https://cdn.getalby-assets.com/alby-extension-website-screen/primat-logo_fire-409917ad.svg

File: src/app/screens/Discover/websites.json [hunk 2]
```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
```
Comment (escapedcat) [inline src/app/screens/Discover/websites.json]: I don't think this subtitle is useful. If there's no good subtitle can it be just empty?

**Review by escapedcat — CHANGES_REQUESTED** [review]

File: src/app/screens/Discover/websites.json (hunk 2 above)
Comment (Tejasp2003) [reply src/app/screens/Discover/websites.json]: All the other websites have a subtitle. The empty subtitle is not looking good, so I added this. Also, There is no subtitle or description on the primal website which we can use, so I put this subtitle.

File: src/app/screens/Discover/websites.json (hunk 1 above)
Comment (Tejasp2003) [reply src/app/screens/Discover/websites.json]: Okay, I will make changes

**Pull request merged**